build/packages.go

```diff
@@ -0,0 +1,31 @@
+package build
+
+import "fmt"
+
+// Binary is an executable package
+type Binary struct {
+	// Path to the binary
+	Path string
+}
+
+// Location of the binary
+func (b *Binary) Location() string {
+	return b.Path
+}
+
+// DockerImage is a container
+type DockerImage struct {
```

Comment:
i know we're unlikely to move away from Docker for now but should this be `ContainerImage` instead?

Reply:
Yeah, I did consider this - i'm happy with either.

Reply:
Just use the word Image. Do not be tied to Docker or Container